addon/components/sl-calendar.js

```diff
@@ -59,13 +49,13 @@ export default Ember.Component.extend({
          * Change the currently-viewed month by incrementing or decrementing
          * the currentMonth (and currentYear if needed)
          *
-         * @function actions.changeMonth
-         * @param    {number} monthMod - A number to adjust the currentMonth by
-         *           (positive to increment, negative to decrement). The
-         *           currentYear is adjusted as needed.
-         * @returns  {void}
+         * @function actions:changeMonth
+         * @param {Number} monthMod - A number to adjust the currentMonth by
```

Comment:
Do we want a "-" separating the param name and the description? Think this will cause the description to then display as "- ..." in the generated documentation.